src/build.py

```diff
@@ -576,26 +576,42 @@ def GetRepoInfo():
   return info
 
 
+def Which(name):
+  """Find an executable on the system by name. If not found return ''."""
+  # If we want to run this on Windows, we'll have to be smarter.
```

Comment:
Hm interesting, I thought check_output worked on Windows. In any case the comment refers to the existence of the `which` program. Windows has a similar thing called `where` although for NaCl we did it manually: https://chromium.googlesource.com/native_client/src/native_client/+/master/pynacl/file_tools.py#72